knowledge/astronomy/constellations/Phoenix/qna.yaml

```diff
@@ -0,0 +1,20 @@
+created_by: Pablo Casal pcasal@gmail.com
+domain: constellation
+seed_examples:
+- answer: The constellation does not lie on the galactic plane of the Milky Way, and there are no prominent star clusters. Lying within the bounds of the constellation is the gigantic Phoenix cluster, which is around 7.3 million light years wide and 5.7 billion light years away. It is located in the southern sky. The constellation stretches from roughly −39° to −57° declination, and from 23.5h to 2.5h of right ascension. The constellations Phoenix, Grus, Pavo and Tucana, are known as the Southern Birds.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!